### ISSUE TITLE
Allow multiple automates to add tags on single file

### ISSUE THREAD

**Issue by croontje** [ciur/papermerge#177]
Right now if there is an automation that matches the file it gets executed. So if I have an automation that searches for "Invoice" and that adds the tag "Invoice" the file gets moved and the matching stops.

But imagine I have 3 kinds of documents: "Invoice", "Contract", "other"
And 20 companies

For each of those kinds of documents I have to make automations for all the companies: 20*3 = 60
But if you make it possible to have multiple automations to match a single file you can make an automation to tag the document types, and one fr each companyname: 20+3=23

And imagine it would be possible to have an automation that tags a file with the year: 2020, 2021, ...
You can add a single extra automation and tag each of the files with an "year" tag. 

(of course only 1 automation can move a file)

**Comment by dani** [comment]
Yes, this would be a great enhancement ! Destination folder could be optional for automates, new documents would flow through all the automates until one which has a destination folder set is met. Ordering automates would be needed though.

**Comment by amo13** [comment]
As of now, I think this is semi-possible already. I can confirm that the documents which I drop into the inbox match multiple automations and get the respective distinct tags. The only thing is that I can't say beforehand where they will be moved if the automations instruct to move the document to different placed.

Tied to #196

**Comment by ajarzyn** [comment]
As @dani mentioned "Ordering automates would be needed though."

In my opinion that would be best solution to prioritize automation easily making linked list from them.
+1 on priority on automation from me.

Maybe it would be better to open "Feature request" for this functionality alone?
